.github/workflows/workflow.yaml

```diff
@@ -45,14 +45,12 @@ jobs:
           check-latest: true
       - run: yarn
       - run: yarn test:ci
-  #      - name: Upload Coverage Report
-  #        uses: actions/upload-artifact@v3
-  #        with:
-  #          path: coverage/lcov-report
-  #      - name: 'Upload coverage to Codecov'
-  #        uses: codecov/codecov-action@v3
-  #        with:
-  #          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
The token is not required since it's a public repo and it's uploaded from GH actions.

Reply:
Separate to this PR, should we update typescript-app-starter to remove the token since we no longer use it for private repos?

Reply:
I didn't realize we don't need codecov token for public repos. That's great

Reply:
I've updated the instructions on the app starter: https://github.com/valora-inc/typescript-app-starter/pull/39